ui/app/components/secure-variable-form.js

```diff
@@ -2,12 +2,17 @@ import Component from '@glimmer/component';
 import { action } from '@ember/object';
 import { tracked } from '@glimmer/tracking';
 import { inject as service } from '@ember/service';
+import { trimPath } from '../helpers/trim-path';
+
 export default class SecureVariableFormComponent extends Component {
   @service router;
+  @service store;
 
   @tracked
   shouldHideValues = true;
 
+  @tracked duplicatePathError = false;
```

Comment:
`nit` `non-blocking`:  This has a union type of `boolean` and `interface {variable: Promise<Array<EmberDataObject>>}`.

Slightly confusing if I'm scanning this. But not a blocker by any means at all. I'm usually just documenting my learning when I do review.

Reply:
Good call; let me set default to null here and name it better.